### ISSUE TITLE
twisted.conch.endpoints should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8722]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8722|
|Type|enhancement|
|Created|2016-08-03 14:51:43Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8722 8722
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470235903614234 1470235903614234
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)<a name="note_1"></a>|@einarfd commented|
|-|-|

Pull request that fixes this can be found at https://github.com/twisted/twisted/pull/456

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

In changeset 5dd584f76cd9fabb048d574e4913dcdd8edab50b
```
#!CommitTicketReference repository="" revision="5dd584f76cd9fabb048d574e4913dcdd8edab50b"
Adding topfile for conch.endpoints, bug 8722
```

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

In changeset 43ac6ba2bf21c2e8aada569926c83e7286a54c53
```
#!CommitTicketReference repository="" revision="43ac6ba2bf21c2e8aada569926c83e7286a54c53"
Adding topfile for conch.endpoints, bug 8722
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset c5da63efa363fb30f4fd60af0c36ebcce2b2cdd4
```
#!CommitTicketReference repository="" revision="c5da63efa363fb30f4fd60af0c36ebcce2b2cdd4"
Merge pull request #457 from twisted/8722-einarfd-py3-conch-endpoints

Author: einarfd
Reviewer: rodrigc
Fixes: #8722 

Port twisted.conch.endpoints to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
